### PR TITLE
[Changelog] Update for Hammer-10

### PR DESCRIPTION
**Pull request by JPrause**
[skip ci]

### CODE REVIEW

**Comment by JPrause** [comment]
@miq-bot add_label documentation

**Comment by JPrause** [comment]
@miq-bot add_label hammer/yes

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,15 +5,19 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
 
-## Ivanchuk Beta1
+## Hammer-10
+
+### Fixed
+- Don't set User.current_user [(#293)](https://github.com/ManageIQ/manageiq-automation_engine/pull/293)
```
Comment (gmcculloug) [inline CHANGELOG.md]: @JPrause I feel a more accurate title for this PR would be `Replace "User.current_user" with "User.with_user"` 

@tinaafitz Are you ok with this?  Should we change the PR title as well?

Comment (gmcculloug) [reply CHANGELOG.md]: Sorry @JPrause I wanted to title to read as the whole string below:

`Replace User.current_user with User.with_user`

I removed the quotes this time.  Also updated the PR to match.

Comment (JPrause) [reply CHANGELOG.md]: No worries,...I guess a literal guy. Ha! Will update now.

Comment (JPrause) [reply CHANGELOG.md]: and I went over to the PR to make sure I got it right. :smile:

Comment (gmcculloug) [reply CHANGELOG.md]: I wondering if I did that with you in mind.  🤔 

Thanks for the updates @JPrause

**Comment by gmcculloug** [comment]
@bdunne Are we using `[skip_ci]` in the description properly?  The running Travis test would indicate maybe we aren't.

File: CHANGELOG.md (hunk 1 above)
Comment (tinaafitz) [reply CHANGELOG.md]: @gmcculloug yes, that makes sense. I saw that PR title was updated.

**Comment by simaishi** [comment]
Hammer backport details:

```
$ git log -1
commit 771d999e4bcf2dc99c742734de313f11c5b19913
Author: Greg McCullough <gmccullo@redhat.com>
Date:   Fri Aug 9 15:38:42 2019 -0400

    Merge pull request #347 from JPrause/hammer10_cl
    
    [Changelog] Update for Hammer-10
    
    (cherry picked from commit 77ebffd419ce07b99c8b6d5b4813848fa97a1e51)
```

**Comment by bdunne** [comment]
> @bdunne Are we using `[skip_ci]` in the description properly? The running Travis test would indicate maybe we aren't.

https://docs.travis-ci.com/user/customizing-the-build/#skipping-a-build

**Pull request merged**